en/log/flight_review.md

```diff
@@ -0,0 +1,332 @@
+Note that this allows us to considerably increase the cutoff frequency of the software filters
+
+![Low vibration QAV-R 5 Racer - FFT plot](../../assets/flight_log_analysis/flight_review/vibrations_good_actuator_controls_fft.png)
+
+DJI F450 frame (good vibration).
+Note the blade passing frequency of the propellers around 100 Hz.
+
+![Low vibration DJI F450 - FFT plot](../../assets/flight_log_analysis/flight_review/vibrations_f450_actuator_controls_fft.png)
+
+S500 frame:
+
+![Low vibration S500 actuator controls - FFT plot](../../assets/flight_log_analysis/flight_review/vibrations_s500_actuator_controls_fft.png)
+
+> **Note** While the plot above looks good, the [Raw Acceleration graph for the same flight](#raw_acc_s500) shows that the vibration levels are a bit high for x and y.
+  This is a good example of why it is worth checking several graphs!
+
+#### Examples: Bad Vibration
+
+This example shows a peak in frequency close to 50 Hz. 
+In this case the cause was a landing gear that had a bit of play.
+
+![Vibrations in landing gear - FFT plot](../../assets/flight_log_analysis/flight_review/vibrations_landing_gear_actuator_controls_fft.png)
+
+
+### Acceleration Power Spectral Density
+
+This is a 2D frequency plot showing the frequency response of the raw accelerometer data over time (it displays the sum for the x, y and z axis).
+The more yellow an area is, the higher the frequency response at that time and frequency.
+
+Ideally only the lowest part up to a few Hz is yellow, and the rest is mostly green or blue.
+
+
+#### Examples: Good Vibration
+
+[QAV-R 5" Racer](https://docs.px4.io/en/frames_multicopter/qav_r_5_kiss_esc_racer.html) frame (excellent vibration).
+
+![Low vibration QAV-R 5 Racer - spectral density plot](../../assets/flight_log_analysis/flight_review/vibrations_good_spectral.png)
+<!-- https://logs.px4.io/plot_app?log=cd88b091-ec89-457c-85f6-e63e4fa0f51d -->
+
+DJI F450 frame (good vibration).
+Note the blade passing frequency of the propellers around 100 Hz.
+![Low vibration DJI F450 - spectral density plot](../../assets/flight_log_analysis/flight_review/vibrations_f450_spectral.png)
+
+S500 frame: <!-- the Actuator Controls FFT plot looks good, whereas the vibration levels are a bit high for x and y in the Raw Acceleration plot (which is typical for an S500 airframe). This is at the limit where it starts to negatively affect flight performance.
+GOOD or bad?
+-->
+
+![Vibration S500 - spectral density plot](../../assets/flight_log_analysis/flight_review/vibrations_s500_spectral.png)
+
+
+#### Examples: Bad Vibration
+
+? <!-- In this log the vibration levels are too high, apparent in the Raw Acceleration plot - the graph of the z-axis overlaps with the x/y-axis graph: -->
+
+
+![High vibration in spectral density plot](../../assets/flight_log_analysis/flight_review/vibrations_too_high_spectral.png)
+
+This example shows a peak in frequency close to 50 Hz. 
```

Comment:
So is it the fact that this is up around 50Hz that makes it a problem? Why is this a problem but not the 100Hz propeller stuff you pointed to above?

Reply:
The problems are:
- it's a single frequency, and it's very strong
- the problem is that it's quite low, close to the vehicle dynamics: with the default filter settings of 80 Hz, the 100 Hz are filtered out (or at least dampened), but not vibrations at 50 Hz.

Reply:
Note, this is generally what you want:
- Look, see what the plot shows
- What might cause this
- Why is it a problem